src/main/cpp/moduledrivers/ConnectorX.cpp

```diff
@@ -63,15 +64,15 @@ uint16_t ConnectorX::ConnectorXBoard::readAnalogPin(AnalogPort port) {
 void ConnectorX::ConnectorXBoard::setLedPort(LedPort port) {
   ConsoleLogger::getInstance().logInfo("requested led port", (int)port);
 
-  if (port != _currentLedPort) {
+  // if (port != _currentLedPort) {
```

Comment:
Does this just disable port 0 entirely?

Reply:
no, it only bypasses the cache check to always set the requested port

Reply:
The Rio is still so fast that the Pico is unable to keep up with two I2C transmissions in-a-row